string sensor_name
Operation data
---
